python/sglang/srt/models/deepseek_v2.py

```diff
@@ -353,6 +342,18 @@ def __init__(
             ),
         )
 
+        self.topk = TopK(
```

Comment:
Can we move self.topk back to above?

Reply:
It needs experts to be initialized first now